XenModel/Messages.resx

```diff
@@ -5266,6 +5266,9 @@ Warning: to prevent data loss you must ensure that the LUN is not in use by any
   <data name="EXPORT_AS_BACKUP" xml:space="preserve">
     <value>Export to File...</value>
   </data>
+  <data name="EXPORT_ERROR_GREATER_THAN_2TB_OVA_OVF" xml:space="preserve">
+    <value>VMs with disks greater than 2TB cannot be exported as an OVF/OVA package.</value>

```

Comment:
The message here says 2TB, but the code in the select VMs page is cheking against 2TiB. Can we confirm which of the two it is, TB or TiB, and modify either the message or the code?

Also, let's not hardcode the value in the message, but rather use the placeholder `{0}`; we can use the Utils class methods to get the disk size string from a number . We can even define the constant DISK_MAX_SIZE_FOR_OVF_EXPORT in the ExportSelectVMsPage.cs so that the number is in one place that can be easily updated  if the max size changes in future.

Reply:
Just checked and it is 2 TiB that we want.